migration-tools/migrate-rosdistro.py

```diff
@@ -80,13 +80,15 @@ def write_tracks_file(tracks, commit_msg=None):
 repositories_to_retry = []
 for repo_name, repo_data in sorted(source_distribution.repositories.items()):
     if repo_name not in dest_distribution.repositories:
-        new_repositories.append(repo_name)
         dest_repo_data = copy.deepcopy(repo_data)
-        release_tag = dest_repo_data.release_repository.tags['release']
-        release_tag = release_tag.replace(args.source,args.dest)
-        dest_repo_data.release_repository.tags['release'] = release_tag
+        if dest_repo_data.release_repository:
+            new_repositories.append(repo_name)
+            release_tag = dest_repo_data.release_repository.tags['release']
+            release_tag = release_tag.replace(args.source,args.dest)
+            dest_repo_data.release_repository.tags['release'] = release_tag
         dest_distribution.repositories[repo_name] = dest_repo_data
-    elif dest_distribution.repositories[repo_name].release_repository.version is None:
+    elif dest_distribution.repositories[repo_name].release_repository is not None and \
```

Comment:
Why `is not None` here versus just pythonic check on line 84?

Reply:
I prefer explicit `is not None` in compound conditionals but I'm easily swayed.